httpev.ml

```diff
@@ -1119,6 +1123,24 @@ let setup_lwt config answer =
 let server_lwt config answer =
   Lwt_main.run @@ setup_lwt config answer
 
+let setup_bind_lwt addr config answer =
```

Comment:
there is `config.connection` which this api totally ignores which is not cool. Instead we should break api and make config.connection a variant itself and make all setup_* functions honor that variant and provide convenience functions to set that variant